Career Endorsement Tracker/Career Endorsement Tracker/ViewControllers/Home/HomeDetailTableViewCell.swift

```diff
@@ -20,23 +20,29 @@ class HomeDetailTableViewCell: UITableViewCell {
             updateViews()
         }
     }
+    let completedImage = UIImage(named: "checkBox")
+    let incompleteImage = UIImage(named: "uncheckedBox")
     
     private func updateViews() {
         guard let step = step else {
             print("No step")
             return
         }
         stepTextView.text = step.steps_description
+        stepTextView.sizeToFit()
         if step.is_complete {
             submitCompletionButton.backgroundColor = .green
-            submitCompletionButton.setTitle("Completed", for: .normal)
+            submitCompletionButton.setImage(completedImage, for: .normal)
```

Comment:
The checked button does not appear.  It is functional, but does not appear.  This UI change is incomplete.  This must be resolved before this PR is merged.